bundle.go

```diff
@@ -176,6 +176,9 @@ func (r *DownloadRequest) downloadBundle() error {
 
 	if err != nil {
 		log.Errorf("Unable to download blob file blobId=%s err:%v", r.blobId, err)
+		if downloadedFile != "" && os.Remove(downloadedFile) != nil {
```

Comment:
Just a nitpick. Removing stale files could be done in the background (go routine), as the files could be big & hence slow. But, not a big deal.